apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/util/checker/AuthUserPasswordChecker.java

```diff
@@ -58,7 +65,13 @@ private boolean isCommonlyUsed(String password) {
     if (Strings.isNullOrEmpty(password)) {
       return true;
     }
-    for (String s : LIST_OF_CODE_FRAGMENT) {
+    List<String> listOfCodeFragment;
+    if (portalConfig != null && !portalConfig.listOFCodeFragment().isEmpty()) {
+      listOfCodeFragment = portalConfig.listOFCodeFragment();
+    } else {
+      listOfCodeFragment = DEFAULT_LIST_OF_CODE_FRAGMENT;
+    }
```

Comment:
portalConfig won't be null, so how about we do like this?
```suggestion
    List<String> listOfCodeFragment = portalConfig.listOFCodeFragment();
    if (listOfCodeFragment.isEmpty()) {
      listOfCodeFragment = DEFAULT_LIST_OF_CODE_FRAGMENT;
    }
```

Reply:
Done.